starter-code/index.html

```diff
@@ -3,40 +3,83 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+
+<header>
+ <img class="jump-spot" src="images/spotify-logo.png">
+  <ul>
+    <li><a class="greenG" href="#0">Premium</a></li> 
+    <li><a class="greenG" href="#0">Discover</a></li> 
+    <li><a class="greenG" href="#0">Help</a></li> 
+    <li><a class="greenG" href="#0">Download</a></li>
+  </ul>
+</header>
+
+<div class="subheader">
+  <h1>Music for everyone.</h1>
+  <p>Spotify is now free on mobile, tablet and computer.<br>
+  Listen to the right music, wherever you are.</p>
+</div>
+
+<div class="what-is">
+  <h2>What’s on Spotify?</h2>
+
+  <div class="orga">
+    <div class="col">
+    <img class="grow" src="images/music-icon.png">
```

Comment:
you don't use the class grow in your css. It's better to remove classes that you don't use because  it makes it easier to read the HTML